test/external/external_test_onnx_backend.py

```diff
@@ -206,6 +214,37 @@ def supports_device(cls, device: str) -> bool:
   backend_test.include('test_zfnet512')
   backend_test.include('test_vgg19')
 
+if getenv('CLANG') and CI:
```

Comment:
Why do they crash? That's probably out of bounds memory accesses, and bugs in this PR. We can't just skip the tests

Reply:
it was because of a bad division by zero because the preventions we do in `sign` only work for floats... updated sign to work for all dtypes
https://github.com/tinygrad/tinygrad/pull/2264/files#diff-7703e7b63c9844f019ae3609629fd9c9bab290d11ae6608ed2d09edd49bab2dbR631